### ISSUE TITLE
Include images in article summaries

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/ai-experiments#195]
When summarizing an article, also incorporate alt text from images

If an image doesn't have (good) alt text, create some on the fly to incorporate that in the summary as well.
